docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -114,36 +125,35 @@ The following output appears:
   validator, simply remove all files from /var/lib/sawtooth.
 
 
-Start Validator
----------------
+Starting the Validator
+----------------------
 
 To start a validator that listens locally on the default ports, run the
-following commands from a Linux terminal:
+following commands from the validator window:
```

Comment:
I think "from a Linux terminal" was correct here. The terminal doesn't really become a "validator window" until the `sudo -u sawtooth sawtooth-validator -vv` command is run (which starts up the validator in that terminal)

Reply:
@dplumb94 - Thanks for this catch!  I was trying a new thing that doesn't quite work - i.e, "naming" the current terminal window in the previous step (see line 106).  Yup, it's confusing.  I'd like to fix it by not mentioning "from a Linux terminal", because they just opened the terminal in the previous step and should have a prompt available.  That is, I propose changing this sentence to: 

     "To start a validator that listens locally on the default ports, run the following commands:"

Reply:
That sounds like a good idea